modules/core/src/copy.cpp

```diff
@@ -320,14 +320,17 @@ void Mat::copyTo( OutputArray _dst ) const
             size_t len = sz.width*elemSize();
 
 #if defined HAVE_IPP
-            CV_IPP_CHECK()
+            if (len == (unsigned int)(int)len)
```

Comment:
This check is not enough. Issue`s sample will not pass. `sz.height` value is critical (=128)